README.md

```diff
@@ -599,8 +599,9 @@ cdnvm() {
         fi
     fi
 }
+
+cdnvm "$PWD" || exit
```

Comment:
is the intention here that if cdnvm fails, the alias should not be set?

Reply:
Yes, this was meant to address https://www.shellcheck.net/wiki/SC2164, but I think it's surprising with the reordering which was meant to keep the `cdnvm` code together and the alias after and separate since shellcheck advised against using an alias in the same parsing unit: https://www.shellcheck.net/wiki/SC2262

To adhere more to the principle of least surprise, I'm going to reorder it back and still have the exit:
```sh
# ...
alias cd='cdnvm'
cdnvm "$PWD" || exit
# ...
```